atomic_reactor/koji_util.py

```diff
@@ -176,3 +177,13 @@ def get_koji_task_owner(session, task_id, default=""):
     else:
         koji_task_owner = default
     return koji_task_owner
+
+
+def get_koji_session_and_check(workflow, fallback):
```

Comment:
Adding a comment here for tracking purposes. As per IRC chat:

- `get_koji` should be performing check on `koji_principal` and `koji_keytab`; technically only needed when fallback is used since schema enforces the check, but harmless to perform in both cases.
- `get_koji_session_and_check` should be removed. `get_koji_session` will use `get_koji` which already contains check

Reply:
ok